MiCCDataAnalytics/PowerBI/Configuration/Readme.md

```diff
@@ -0,0 +1,63 @@
+# Readonly Configuration
+This repository contains Power BI templates providing a readonly view of configuration
```

Comment:
grammer police :P

should be 
read-only

Reply:
done